adapter/src/main/java/org/apache/calcite/rel/rules/materialize/KuduMaterializedViewRule.java

```diff
@@ -0,0 +1,1854 @@
+/* Copyright 2022 Twilio, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.calcite.rel.rules.materialize;
```

Comment:
This is just a copy paste from here https://github.com/apache/calcite/pull/2876. Its easier tor review the changes in the calcite PR.